packages/service-client/odsp-client/.gitignore

```diff
@@ -0,0 +1,52 @@
+# Compiled TypeScript and CSS
```

Comment:
We can probably minimize this. The repo-root gitignore should cover most of this.